distribution/build.gradle

```diff
@@ -43,6 +46,21 @@ buildscript {
 Collection distributions = project.subprojects.findAll {
    it.path.contains(':tools') == false && it.path.contains(':bwc') == false }
 
+/*****************************************************************************
+ *                  Third party dependencies report                          *
+ *****************************************************************************/
+
+// Concatenates the dependencies CSV files into a single file
+task generateDependenciesReport(type: ConcatFilesTask) {
+  files = fileTree(dir: project.rootDir,
+          excludes: ['qa/**/dependencies.csv', 'benchmark/**/dependencies.csv', 'test/**/dependencies.csv'],
+          include: '**/dependencies.csv' )
+  headerLine = "name,version,url,license"
+
+  def outputFile = System.getProperty('csv')?: "${project.buildDir}/dependencies/es-dependencies.csv"
+  target = file(Paths.get(outputFile).toFile())
```

Comment:
Why isn't this just `new File(outputFile)`?

Reply:
Fixed